packages/graphql/src/utils/get-relationship-direction.ts

```diff
@@ -45,11 +45,7 @@ function getRelationshipDirection(
     /**
      * Duplicate of the schema-model `getCypherDirection` method;
      **/
-    if (
-        fieldArgs.directed === false ||
-        relationField.queryDirection === RelationshipQueryDirectionOption.UNDIRECTED_ONLY ||
-        relationField.queryDirection === RelationshipQueryDirectionOption.UNDIRECTED
-    ) {
+    if (fieldArgs.directed === false || relationField.queryDirection === RelationshipQueryDirectionOption.UNDIRECTED) {
```

Comment:
I believe the `fieldArgs` is related the the GraphQL argument that is no longer present